ebt/Application.vala

```diff
@@ -102,13 +101,19 @@ public class BluetoothApp : Gtk.Application {
                                     bt_senders.remove_link (bt_senders.find (sender));
                                 }
                             });
+                            check_quit ();
                         });
                     }
                 });
-
                 arg_files = {};
                 send = false;
+            } else {
+                stderr.printf ("\n" + _("Must supply at least one file to send") + "\n\n");
             }
+        } else if (!is_held) {
+            stderr.printf ("\n" + _("Must supply at least one option") + "\n\n");
```

Comment:
This seems incorrect. We usually don't use `stderr` or force newlines or translate terminal output. Is there a non-obvious reason for this here?

Reply:
@danirabbit The reason for using `stderr` is to avoid the terminal prompt interfering with the layout of the output. If you use `warning` then every warning is preceded by a (potentially long) prompt and the output is messy.  You have to have explicit newlines with `stderr.printf` to get the desired layout.  Is there a way of suppressing the prompt using `warning`?

I am in two minds about translating the usage warnings - the option entries in e.g. Files and Terminal are translated which seems reasonable for help information even if in a terminal. I agree that technical output intended mainly for developers should not be translated.